### ISSUE TITLE
problem with timezones witch has munites difference

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#47]
```
the tz_delta(Time t) method in twitter_interface.vala doesn't support
minutes. for example offset of tehran is +0330 but tz_delta returns 3
instead of 3.5.

i have attached a patch that will fix it
```

Original issue reported on code.google.com by `everpl...@gmail.com` on 14 Jan 2010 at 9:21

Attachments:
- [pino-timezone.diff](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-47/comment-0/pino-timezone.diff)

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot. I'm very grateful for your help.
```

Original comment by `tro...@gmail.com` on 14 Jan 2010 at 11:01
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by cb9e2c3a53 revision.
```

Original comment by `tro...@gmail.com` on 27 Jan 2010 at 4:20
- Changed state: **Fixed**
